tutorial/lesson_07_multi_stage_pipelines.cpp

```diff
@@ -30,7 +31,7 @@ int main(int argc, char **argv) {
     // first horizontally, and then vertically.
     {
         // Take a color 8-bit input
-        Image<uint8_t> input = load<uint8_t>("images/rgb.png");
+        Image<uint8_t> input = load<Image<uint8_t>>("images/rgb.png");
```

Comment:
Also, re: WITH_EXCEPTIONS, that's generally set when building Halide itself; we need a signal to use when compiling thing-that-uses-Halide. (We could of course use the same symbol, just pointing out that it's not necessarily set in lockstep how Halide is built.)